spec/components/date_range_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe 'date_range', type: :helper do
+  subject do
+    Nokogiri::HTML.parse(date_range)
+  end
+
+  let(:form) do
+    helper.bootstrap_form_for :foo, url: '/' do |f|
+      return f
```

Comment:
The intention was to escape the form's block, returning the form object to the calling scope. The thing is, that the calling scope is itself a block, so the return should break out of this too, should it? So, now that you ask, I really cannot explain why this actually works...